src/app/services/api-users-crud.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { UsersObject } from '../../../api-objects/UsersObject';
+import { ServerApiService } from './server-api.service';
+import { Response } from '../../../api-objects/GenericResponse';
+
+@Injectable()
+export class ApiUsersCrudService {
+
+  constructor(private serverApiService: ServerApiService) { }
+
+  // Response data should be the generated password.
+  public createUser(obj: UsersObject): Observable<Response<string>> {
```

Comment:
I'd rename the variable so something more specific.
```suggestion
  public createUser(userObject: UsersObject): Observable<Response<string>> {
```